cumulusci/tasks/apex/batch.py

```diff
@@ -33,9 +32,28 @@ def _run_task(self):
 
         self.logger.info("Job is complete.")
 
+        vals = {
+            key: value
+            for key, value in self.batch.items()
+            if key
+            in [
+                "Id",
+                "Status",
+                "ExtendedStatus",
+                "NumberOfErrors",
+                "JobItemsProcessed",
+                "TotalJobItems",
+            ]
+        }
         if not self.success:
-            self.logger.info("There were some batch failures.")
-            raise SalesforceException(self.batch["ExtendedStatus"])
+            self.logger.info("There have been some batch failures.")
+            self.logger.info("Error values:")
+            self.logger.info(repr(vals))
+            raise SalesforceException("There were import errors", vals)
+        elif not self.done_for_sure:
+            self.logger.info("The final record counts do not add up.")
+            self.logger.info("This is probably related to W-1132237")
```

Comment:
We should probably link to the known issue (which is public) rather than an internal work item.

Reply:
Google turns up the public page when you search for W-1132237

Reply:
I forgot those are referenced publicly. But why not save someone the googling and link to https://success.salesforce.com/issues_view?id=a1p300000008XvaAAE

Reply:
99f6708a